tests/e2e/singlecard/test_chunked.py

```diff
@@ -34,34 +34,49 @@ def test_models(
     max_tokens: int,
     monkeypatch: pytest.MonkeyPatch,
 ) -> None:
-    return
-
     prompts = "The president of the United States is"
 
     sampling_params = SamplingParams(
         max_tokens=max_tokens,
         temperature=0.0,
     )
 
-    vllm_model = LLM(model, long_prefill_token_threshold=4, enforce_eager=True)
-    output_chunked = vllm_model.generate(prompts, sampling_params)
-    logprobs_chunked = output_chunked.outputs[0].logprobs
+    vllm_model = LLM(model,
```

Comment:
Thanks for this change, can you use VllmRunner instead?https://github.com/vllm-project/vllm-ascend/blob/2da281ec5a0dccd2d4bbdd7fb88724c65d094f80/tests/e2e/conftest.py#L81 

It can cleanup resouce by `__exit__` func auomaticlly.

Reply:
> Thanks for this change, can you use VllmRunner instead?
> 
> https://github.com/vllm-project/vllm-ascend/blob/2da281ec5a0dccd2d4bbdd7fb88724c65d094f80/tests/e2e/conftest.py#L81
> 
> It can cleanup resouce by `__exit__` func auomaticlly.

Nice!!!